modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCommerceDestinationNamesCheck.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.StringUtil;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Michael Cavalcanti
+ */
+public class UpgradeJavaCommerceDestinationNamesCheck
+	extends BaseUpgradeMatcherReplacementCheck {
+
+	@Override
+	protected String formatMatcherIteration(
+		String content, String newContent, Matcher matcher) {
+
+		String constant = matcher.group(1);
+
+		if (_deprecatedConstants.contains(constant)) {
+			String constantCall = matcher.group();
+
+			String newConstantCall = constantCall;
```

Comment:
@michaelmdelcavalcanti, this variable isn't doing anything, I'm right? Can we remove it?

```

			return StringUtil.replace(
				newContent, constantCall,
				StringUtil.replace(
					constantCall, constant, "COMMERCE_" + constant));
```

Reply:
Done